src/screens/TransactionViews/TransactionView.res

```diff
@@ -92,25 +88,26 @@ let make = (~entity=TransactionViewTypes.Orders) => {
     }
   }
 
-  let setActiveViewOnLoad = () => {
-    let appliedStatusFilter =
-      filterValueJson->JSON.Encode.object->getDictFromJsonObject->getArrayFromDict("status", [])
+  let settingActiveView = entity => {
+    let customFilterKey = getCustomFilterKey(entity)
```

Comment:
define customFilterKey globally to this module

Reply:
okay